ml-agents/mlagents/trainers/trainer.py

```diff
@@ -32,11 +33,18 @@ def __init__(self, brain, trainer_parameters, training, run_id):
         self.brain_name = brain.brain_name
         self.run_id = run_id
         self.trainer_parameters = trainer_parameters
+        self.time_start_experience_collection = None
+        self.delta_last_experience_collection = None
+        self.cumulative_returns_since_policy_update = [0]
```

Comment:
Why is this a zero again ?

Reply:
This was just intended as an explicit initialization value. Thinking about it, this creates a problem where the mean will lower than expected, will fix.